apps/desktop-wallet/src/api/apiDataHooks/utils/useFetchFtList.ts

```diff
@@ -35,17 +34,21 @@ type FTListProps = {
 }
 
 const useFetchFtList = (props?: FTListProps): FTList => {
-  const networkId = useAppSelector((s) => s.network.settings.networkId)
+  const networkId = useAppSelector(selectCurrentlyOnlineNetworkId)
   const network = networkId === 0 ? 'mainnet' : networkId === 1 ? 'testnet' : undefined
 
   const { data, isLoading } = useQuery({
-    queryKey: [TOKEN_LIST_QUERY_KEY, network],
+    queryKey: ['tokenList', network],
+    staleTime: ONE_DAY_MS,
+    // The token list is essential for the whole app so we don't want to ever delete it. Even if we set a lower gcTime,
+    // it will never become inactive (since it's always used by a mount component).
+    gcTime: Infinity,
```

Comment:
I added comments above each use of `gcTime` so that it's clear now and in the future as to why we chose these values.